packages/forma-36-codemod/bin/cli.js

```diff
@@ -0,0 +1,191 @@
+const path = require('path');
+const chalk = require('chalk');
+const execa = require('execa');
+const isGitClean = require('is-git-clean');
+const inquirer = require('inquirer');
+const meow = require('meow');
+const globby = require('globby');
+const inquirerChoices = require('./inquirer-choices');
+
+const transformerDirectory = path.join(__dirname, '../', 'transforms');
+const jscodeshiftExecutable = require.resolve('.bin/jscodeshift');
+
+function checkGitStatus(force) {
+  let clean = false;
+  try {
+    clean = isGitClean.sync(process.cwd());
+  } catch (err) {
+    if (err && err.stderr && err.stderr.indexOf('Not a git repository') >= 0) {
+      if (!force) {
+        console.log(
+          chalk.yellow(
+            'Not a git repository, you may use the --force flag to override this check',
+          ),
+        );
+        process.exit(1);
+      }
+      clean = true;
+    }
+  }
+
+  if (!clean) {
+    console.log(
+      chalk.yellow(
+        'Before we continue, please stash or commit your git changes.',
+      ),
+    );
+    process.exit(1);
+  }
+}
+
+function runTransform({ files, flags, parser, transformer }) {
+  const transformerPath = path.join(transformerDirectory, `${transformer}.js`);
+
+  let args = [];
+
+  const { dry, print, explicitRequire } = flags;
+
+  if (dry) {
+    args.push('--dry');
+  }
+  if (print) {
+    args.push('--print');
+  }
+
+  if (explicitRequire === 'false') {
+    args.push('--explicit-require=false');
+  }
+
+  args.push('--verbose=2');
+
+  args.push('--ignore-pattern=**/node_modules/**');
+
+  args.push('--parser', parser);
+
+  if (parser === 'tsx') {
+    args.push('--extensions=tsx,ts,jsx,js');
+  } else {
+    args.push('--extensions=jsx,js');
+  }
+
+  args = args.concat(['--transform', transformerPath]);
+
+  if (flags.jscodeshift) {
+    args = args.concat(flags.jscodeshift);
+  }
+
+  args = args.concat(files);
+
+  console.log(`Executing command: jscodeshift ${args.join(' ')}`);
+
+  const result = execa.sync(jscodeshiftExecutable, args, {
+    stdio: 'inherit',
+    stripEof: false,
+  });
+
+  if (result.error) {
+    throw result.error;
+  }
+}
+
+function expandFilePathsIfNeeded(files) {
+  const shouldExpandFiles = files.some((file) => file.includes('*'));
+  return shouldExpandFiles ? globby.sync(files) : files;
+}
+
+function run() {
+  const cli = meow(
+    {
+      description: 'Codemods for updating Forma-36 codes',
```

Comment:
@massao Minor suggestion: I felt it's generic 'Codemods for updating Forma-36 codes'. Can it be more detailed? 
Example: _"Codemods for updating new color tokens in Forma-36 code"_

Reply:
@anilkk this CLI will be used to apply different transformations, not only color tokens

Reply:
@suevalov oh ok, Got it. Thanks for the clarification.